### PR TITLE
Da 3078

### PR DESCRIPTION
**Pull request by lfpratik**
1. Slack patch no needed, Perceval team reverted to old one.
2. Common repository name hold inside the new alias
3. Reviewer organization query updated

### CODE REVIEW

**Comment by lfpratik** [comment]
@lukaszgryglicki, Just a Note these changes we have to make in da-ds as well

**Comment by lukaszgryglicki** [comment]
I will approve/merge all of this tomorrow, but can you summarize those changes here?

**Comment by lfpratik** [comment]
> I will approve/merge all of this tomorrow, but can you summarize those changes here?

We can sync up on this or I will post changes over slack.

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm
